### DISCUSSION TITLE
Cannot search for map using difficulty ID

### DISCUSSION THREAD

**Discussion by Tanza3D** [ppy/osu#25004] [Ideas] [answered]
on osu!web, if you paste in the difficulty ID, it shows the map;
![image](https://github.com/ppy/osu/assets/33783503/38c20cb7-8b27-4402-b15a-ce1d6ae1db09)
but doing the same in lazer doesn't work
![image](https://github.com/ppy/osu/assets/33783503/b03feb4a-3971-4c09-a0c6-0712176d60de)
I feel having this is a very nice quality of life feature, as it allows you to easily find a beatmap from web inside your game without having to worry about copy pasting the title and getting the mapper right.

**Comment by Bruno5430** [comment, marked as answer]
It actually does, you just had the "featured artist" filter turned on
![image](https://github.com/ppy/osu/assets/111712756/55e9d059-8743-4a21-aa58-90273ca451c8)

**Comment by Tanza3D** [comment]
ohhh... my bad, apologies haha

**Reply by Joehuu** [reply]
We had many reports of this, so something is definitely wrong with the UI. See https://github.com/ppy/osu/issues/22009.
